CHANGELOG.md

```diff
@@ -1,6 +1,245 @@
 Changelog
 =========
 
+1.8.0
+-----
+
+### IMPORTANT NOTES
+
+#### Last release before a PHP minimum version bump
+
+  Release 1.8.0 will be the last release with compatibility for PHP 5.2 - 5.5. With the next release (v2.0.0), the minimum PHP version will be bumped to 5.6+.
```

Comment:
```suggestion
  Release 1.8.0 will be the last release with compatibility for PHP 5.2 - 5.5. With the next release (v2.0.0), the minimum PHP version will be bumped to 5.6.
```

... or 5.6.x ...to prevent giving the impression that it _may_ still become a 7.x version.